Description: Upstream changes introduced in version 1.8.1-6ubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 neko (1.8.1-6ubuntu1) oneiric; urgency=low
 .
   * Fux FTBFS with ld --as-needed. LP: #771060.
 .
 The person named in the Author field signed this changelog entry.
Author: Matthias Klose <doko@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/771060

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- neko-1.8.1.orig/src/tools/install.neko
+++ neko-1.8.1/src/tools/install.neko
@@ -253,9 +253,9 @@ link = function(files,target,params1,par
 		dir[0] + ".o"
 	}));
 	var opt = linkoptions;
-	var cmd = linkcmd+" "+opt+" "+make("-L",libraries);
+	var cmd = linkcmd+" "+make("-L",libraries);
 	var out = " -o ../../bin/"+target+".ndll ";
-	exec(cmd+out+params1+files+params2);
+	exec(cmd+out+files+params1+params2+" "+opt);
 }
 
 // COMPILATION
